lib/bolt/applicator.rb

```diff
@@ -188,6 +184,10 @@ def apply_ast(raw_ast, targets, options, plan_vars = {})
         @executor.await_results(result_promises)
       end
 
+      # Allow for report to exclude event metrics (apply_result doesn't require it to be present)
+      resource_counts = r.ok_set.map { |result| result.event_metrics&.fetch('total') }.compact
```

Comment:
`fetch` will raise a `KeyError` if the key doesn't exist. Is that intended?

Reply:
I think the result produces an error if event_metrics exists but doesn't include 'total'.